lib/devise/models/rememberable.rb

```diff
@@ -66,6 +66,10 @@ def remember_expires_at
         self.class.remember_for.from_now
       end
 
+      def extend_remember_period
```

Comment:
If we remove this method, does the metaprogramming chain that backs these configs will work like the others or we need this delegation for this case?